crates/fdev/Cargo.toml

```diff
@@ -28,7 +28,7 @@ semver = { workspace = true }
 tar = "0.4"
 thiserror = "1"
 tokio = { version = "1", features = ["rt-multi-thread", "sync", "macros", "signal", "parking_lot", "process"] }
-tokio-tungstenite = "0.21.0"
+tokio-tungstenite = "0.20"
```

Comment:
why the regression here?

Reply:
Because stdlib is using 0.20, I will change stdlib to 0.21.